src/utils/logger.ts

```diff
@@ -17,24 +16,22 @@ export class TsLogger extends BaseLogger {
     [LogLevel.fatal]: 'fatal',
   } as const
 
-  public constructor(logLevel: LogLevel, name?: string) {
+  public constructor(logLevel: LogLevel) {
     super(logLevel)
 
-    this.logger = new Logger({
-      name,
-      minLevel: this.logLevel == LogLevel.off ? undefined : this.tsLogLevelMap[this.logLevel],
-      ignoreStackLevels: 5,
-    })
+    this.logger = pino(
+      {
+        name: 'veritable-cloudagent',
+        timestamp: true,
+        level: 'debug',
+      },
+      process.stdout
+    )
   }
 
   private log(level: Exclude<LogLevel, LogLevel.off>, message: string, data?: Record<string, any>): void {
     const tsLogLevel = this.tsLogLevelMap[level]
-
-    if (data) {
-      this.logger[tsLogLevel](message, data)
-    } else {
-      this.logger[tsLogLevel](message)
-    }
+    this.logger[tsLogLevel](message, data || {})
```

Comment:
This isn't going to work as expected because the log functions work differently. You'll have to do something like:

```ts
this.logger[tsLogLevel]('%s %j', message, data || {})
```

Reply:
used, `%o` as per `All arguments supplied after message are serialized and interpolated according to any supplied printf-style placeholders (%s, %d, %o|%O|%j) to form the final output msg value for the JSON log line.` since it's an object. to be fair same output using either, hapy to change to %j, also added some logic since if it's `undefined` was coming up as `no found: undefined` or `not found: {}`